translations/ja.json

```
@@ -555,66 +555,66 @@
+	"Specify an alternate --host-only-cidr value, such as 172.16.0.1/24": "代わりの --host-only-cidr 値を指定します (172.16.0.1/24 など)",
+	"Specify arbitrary flags to pass to the Docker daemon. (format: key=value)": "Docker デーモンに渡す任意のフラグを指定します。(形式: key=value)",
+	"Specify arbitrary flags to pass to the build. (format: key=value)": "ビルドに渡す任意のフラグを指定します。(形式: key=value)",
+	"Specifying extra disks is currently only supported for the following drivers: {{.supported_drivers}}. If you can contribute to add this feature, please create a PR.": "追加ディスク指定は現在 {{.supported_drivers}} ドライバーのみ対応しています。本機能の追加に貢献可能な場合、PR を作成してください。",
+	"StartHost failed, but will try again: {{.error}}": "StartHost に失敗しましたが、再度試してみます: {{.error}}",
+	"Starting control plane node {{.name}} in cluster {{.cluster}}": "{{.cluster}} クラスター中のコントロールプレーンの {{.name}} ノードを起動しています",
+	"Starting minikube without Kubernetes {{.name}} in cluster {{.cluster}}": "{{.cluster}} クラスター中の Kubernetes なしで minikube {{.name}} を起動しています",
+	"Starting tunnel for service {{.service}}.": "{{.service}} サービス用のトンネルを起動しています。",
+	"Starting worker node {{.name}} in cluster {{.cluster}}": "{{.cluster}} クラスター中の {{.name}} ワーカーノードを起動しています",
+	"Starts a local Kubernetes cluster": "ローカルの Kubernetes クラスターを起動します",
+	"Starts a local kubernetes cluster": "ローカルの Kubernetes クラスターを起動します",
+	"Starts a node.": "ノードを起動します。",
+	"Starts an existing stopped node in a cluster.": "クラスター中の既存の停止ノードを起動します。",
+	"Startup with {{.old_driver}} driver failed, trying with alternate driver {{.new_driver}}: {{.error}}": "{{.old_driver}} ドライバーを用いた始動に失敗しましたが、代わりの {{.new_driver}} ドライバーで再試行しています: {{.error}}",
+	"Stopped tunnel for service {{.service}}.": "{{.service}} サービス用トンネルを停止しました。",
+	"Stopping node \"{{.name}}\"  ...": "「{{.name}}」ノードを停止しています...",
+	"Stopping tunnel for service {{.service}}.": "{{.service}} サービスのトンネルを停止しています。",
+	"Stops a local Kubernetes cluster. This command stops the underlying VM or container, but keeps user data intact. The cluster can be started again with the \"start\" command.": "ローカルの Kubernetes クラスターを停止します。このコマンドは下位層の VM またはコンテナーを停止しますが、ユーザーデータは損なわれずに保持します。クラスターは「start」コマンドで再起動できます。",
+	"Stops a node in a cluster.": "クラスター中のノードを停止します。",
+	"Stops a running local Kubernetes cluster": "ローカル Kubernetes クラスターを停止します",
+	"Successfully added {{.name}} to {{.cluster}}!": "{{.cluster}} への {{.name}} 追加に成功しました!",
 	"Successfully deleted all profiles": "全てのプロファイルの削除に成功しました",
-	"Successfully mounted {{.sourcePath}} to {{.destinationPath}}": "",
-	"Successfully purged minikube directory located at - [{{.minikubeDirectory}}]": "",
-	"Successfully started node {{.name}}!": "",
-	"Successfully stopped node {{.name}}": "",
+	"Successfully mounted {{.sourcePath}} to {{.destinationPath}}": "{{.destinationPath}} への {{.sourcePath}} のマウントに成功しました",
+	"Successfully purged minikube directory located at - [{{.minikubeDirectory}}]": "[{{.minikubeDirectory}}] ディレクトリーにある minikube の削除に成功しました",
```

In my previous comment (https://github.com/kubernetes/minikube/pull/13092#discussion_r765461495), I suggested the below. How about it?
```suggestion
	"Successfully purged minikube directory located at - [{{.minikubeDirectory}}]": "[{{.minikubeDirectory}}] にある minikube ディレクトリーの削除に成功しました",
```